docs/pages/cli/faucet.mdx

```diff
@@ -1,3 +1,5 @@
 # mud faucet
 
-**Coming soon**
+This command is deprecated. It can still be used to interact with the deprecated faucet service on the deprecated "lattice testnet", but will be removed in the future.
+
+We recommend to use the [Redstone Holesky testnet](https://redstone.xyz) and new [faucet service](/services/faucet). See [the sample client program](https://github.com/latticexyz/mud/tree/main/examples/faucet-client) for a usage example.
```

Comment:
```suggestion
This command is currently used to interact with the deprecated faucet service on the deprecated "lattice testnet". Once the new Redstone devnet is available, it will be updated to interact with the new [faucet service](/services/faucet).

In the meantime, we recommend to use the [Redstone Holesky testnet](https://redstone.xyz) and self-hosted [faucet service](/services/faucet). See [the sample client program](https://github.com/latticexyz/mud/tree/main/examples/faucet-client) for a usage example.
```